tests/test_blocks.py

```diff
@@ -211,8 +211,7 @@ def test_image_internal_with_caption(mock_download):
     mock_download.return_value = 'image.png'
     pandoc_ast, markdown = process_block(notion_block)
     assert pandoc_ast == Para([
-        Image(('', [], []), [Str('test'), Space(), Str('image')], ('image.png', ''))
-    ])
+        Image(('', [], []), [Str('test'), Space(), Str('image')], ('image.png', ''))])
```

Comment:
In this repo, the convention is to keep the closing brackets on its own line as it was prior to this change.